docs/usage/check.rst.inc

```diff
@@ -90,6 +90,11 @@ The check command verifies the consistency of a repository and the corresponding
 check --repair is a potentially dangerous function and might lead to data loss
 (for kinds of corruption it is not capable of dealing with). BE VERY CAREFUL!
 
+Also note that the check command can report spurious errors when the underlying
+hardware is defective. If you are getting inconsistent error reports then we
+advise you to run hardware testing software such as memory testers to determine
+the exact cause of the problem.
+
```

Comment:
Yes, think we should add something like this, just not sure about the phrasing.

can report -> may report?

inconsistent errors -> check hardware - not sure about the "inconsistent" here. e.g. if one had some bit flipping in RAM and that data was saved to disk, it might as well become permanent and consistently report the same issue again and again.

Reply:
Yea, that's the problem with hardware problems. It's impossible to truly predict how it will influence program behavior.